CONTRIBUTING.md

```diff
@@ -1,14 +1,95 @@
 # Contributors guide
 
-## Markdown style
-
-The specification documents follow the
-[Markdown Style Guide](http://www.cirosantilli.com/markdown-style-guide/). You
-can validate your changes against the guide using
-[remark](https://github.com/remarkjs/remark-lint) which works as a
-[standalone command line tool](https://github.com/remarkjs/remark/tree/master/packages/remark-cli)
-as well as
-[a plugin for various text editors](https://github.com/remarkjs/remark-lint#editor-integrations).
-You can also use [prettier](https://github.com/prettier/prettier) to
-automatically correct some of the style issuse that might be found in the
-proposed changes.
+# Contributing to the BIDS Specification
+
+:tada::balloon::cake: **Welcome to the BIDS Specification repository!** :cake::balloon::tada:
+
+:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+
```

Comment:
hey @Park-Patrick Thank you for writing this up! I liked that the guide felt digestible throughout and easy to understand (especially for those that it maybe their first time navigating GitHub). The example you provided is great! For the 'Making a change with a pull request' I suppose that was more for the starters kit? I think the principle is consistent, just in regards to the specification. I noticed that there were some double bracketed terms (i.e. in 1 (Comment on an existing issue...) the second paragraph of [This blog][dont-push-pull-request]) I wasn't sure if you meant them to linking elsewhere? I think with Kirstie's suggestions (of adding the linter and continuous integration) will make this a nice contributing guide! One thing that maybe considered adding is how the repo is organized (i.e. that the standard is currently in the src folder). I can help with some of this too!

Reply:
good catch! so some of the double bracketed terms had corresponding tags in the starter kit version, but I think some of them became obsolete so i'll either re-link or remove them.

I'm not super familiar with this repo's policies for linter/CI moving forward, so it would be great to get help with that!

Reply:
I see! No worries! 

I can add what I understand from linter/CI as well. I'll work on it from your repo and PR to that